packages/shadydom/tests/shady.html

```diff
@@ -721,6 +731,24 @@
   });
 })
 
+suite('Event property patches should not add new properties', () => {
```

Comment:
It doesn't look like we have tests for "on*" attributes working. Would you mind adding one? We want to ensure that it's running through the polyfill and to do so you should be able to look at `composedPath`.

Reply:
There's a separate file called `event-path.html` that seems to test some of the on* properties:

phases:
https://github.com/webcomponents/polyfills/blob/1aa89fd85ef7f24660233402919a510a1e83aa22/packages/shadydom/tests/event-path.html#L569-L585

targets:
https://github.com/webcomponents/polyfills/blob/1aa89fd85ef7f24660233402919a510a1e83aa22/packages/shadydom/tests/event-path.html#L636-L666

I didn't put these in there because this didn't really seem related to event paths though. Are those the kinds of tests you were looking for? / Should I move these?

Reply:
Sorry, this doesn't make sense. The `on*` attributes are not supported since they are parser driven. Imo, adding support for this is not important so let's skip it.